Text/CustomEntitySearch/words.json

```diff
@@ -0,0 +1,3 @@
+{
+  "targetWords": ["foo1", "foo2"]
```

Comment:
Since the file only contains a list of words, the outer object is not necessary, you can have the array at the top level directly. Or we could make it even easier for the customer and have a text file with one word per line. Less verbose.

Reply:
Currently, the words can include new line characters. Do we not want to support multi-line entities?

Reply:
Let's assume not, at least for now.